src/DIRAC/Resources/Computing/PilotBundle.py

```diff
@@ -44,8 +38,8 @@ def bundleProxy(executableFile, proxy):
 try:
   workingDirectory = tempfile.mkdtemp(suffix='_wrapper', prefix='TORQUE_')
   os.chdir(workingDirectory)
-  open('proxy', "w").write(bz2.decompress(base64.decodestring("%(compressedAndEncodedProxy)s")))
-  open('%(executable)s', "w").write(bz2.decompress(base64.decodestring("%(compressedAndEncodedExecutable)s")))
+  open('proxy', "w").write(bz2.decompress(base64.b64decode("%(compressedAndEncodedProxy)s")))
+  open('%(executable)s', "w").write(bz2.decompress(base64.b64decode("%(compressedAndEncodedExecutable)s")))
```

Comment:
IIRC we agreed at the BiLD that this change is fine even if it isn't tests as it without it there will be issues if the host has a Python 3.9 installation (even if that's rare at the moment).